File: output/llama3.1-8b/avatar/Java/Python/codeforces_99_A.py
import re

def main():
    s = input()
    dot_index = s.find('.')

    if dot_index != -1 and s[dot_index-1] == '9':
        print("GOTO Vasilisa.")
    elif int(s[dot_index+1:]) > 4:
        _str = s[:dot_index]
        print(int(_str) + 1)
    else:
        print(s[:dot_index])

if __name__ == "__main__":
    main()

File: output/gemma4-e2b_IO_1/avatar/Python/Java/codeforces_99_A.java
import java.util.Scanner;

public class codeforces_99_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // Read the input line l
        if (!scanner.hasNextLine()) {
            scanner.close();
            return;
        }
        String l = scanner.nextLine();
        
        // l = l.split('.')
        String[] parts = l.split("\\.");
        
        // s is the first part (l[0])
        String s_str = parts[0];
        
        // p is the second part (l[1])
        String p_str = parts[1];
        
        // i = int(p[0])
        // p_str.charAt(0) gets the first character of the second part
        int i = Character.getNumericValue(p_str.charAt(0));
        
        // Check s[len(s) - 1]
        char lastSChar = s_str.charAt(s_str.length() - 1);
        
        if (lastSChar == '9') {
            System.out.println("GOTO Vasilisa.");
        } else if (lastSChar != '9' && i < 5) {
            // s = "".join(s)
